packages/esm-patient-attachments-app/src/camera-media-uploader/camera-media-uploader.component.tsx

```diff
@@ -115,7 +121,7 @@ const CameraMediaUploadTabs = () => {
 
   return (
     <div className={styles.cameraSection}>
-      <ModalHeader closeModal={closeModal} title={t('addAttachment_title', 'Add Attachment')} />
+      <ModalHeader closeModal={closeModal} title={title ? title : t('addAttachment_title', 'Add Attachment')} />
```

Comment:
You can make this simple as `title || t('addAttachment_title', 'Add Attachment')`